src/App.js

```diff
@@ -0,0 +1,161 @@
+import "macro-css";
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { MainTemperatureBlock } from "./components/DateTemperature/MainTemperatureBlock/index.js";
+import { SearchToolbar } from "./components/SearchToolbar";
+import { DayTemperatureBlock } from "./components/DayTemperature";
+import { TEMPERATURE_INDEXES_TEMPLATE } from "./utilities/constants/temperature-indexes-template";
+import { THEMES } from "./utilities/constants/theme.js";
+import './index.scss';
+
+function App() {
+  const [data, setData] = useState({
+    city: "Vinnytsya",
+    country: "",
+    list: [],
+    days: [],
+    theme: THEMES.light,
+    current: {
+      icon: "",
+      temp: "",
+      text: "",
+    }
+  });
+
+  useEffect(() => {
+    const fetch = async () => {
```

Comment:
move function from useEffect just to component scope. what mean fetch? fetch of what? all handlers or actions should names by an action name: getWeatherData, deleteUserById, handleInputChange etc